UKHO.FmEssFssMock.API/UKHO.FmEssFssMock.API/Enums/Batch.cs

```diff
@@ -74,7 +74,7 @@ public enum Batch
         EssEmptyBatch = 23,
 
         [Description("d0635e6c-81ae-4acb-9129-1a69f9ee58d2")]
-        EssEmptyBatchs57 = 24
+        BesEmptyBatch = 24
```

Comment:
Use Bess to make it consistent in mock as well wherever it get used.
```suggestion
        BeesEmptyBatch = 24
```

Reply:
Resolved